src/tango_sdp_subarray/SDPSubarray/SDPSubarray.py

```diff
@@ -230,6 +232,16 @@ def Configure(self, pb_config, schema_path=None):
 
         pb_config = json.loads(pb_config)
         validate(pb_config, schema)
+
+        for txn in db_config.Config().txn():
+            confdata = pb_config['configure']
+            pb_id = txn.new_processing_block_id(confdata['id'])
```

Comment:
The "type" argument to `new_processing_block_id` should be either "batch" or "realtime". The two options here are really:
1. Just hardcode "realtime" for the moment
2. Don't bother with calling `new_processing_block_id` at all and just use `confdata['id']` for `pb_id` (that's what I expected!). Disadvantage here is that things will go wrong if Configure gets called again with the same id (which - to be fair - should be an error anyway).